components/x-teaser-timeline/src/lib/transform.js

```diff
@@ -19,7 +19,11 @@ const groupItemsByLocalisedDate = (
 
 	items.forEach((item, index) => {
```

Comment:
**question:** I can't see `groupItemsByLocalisedDate` being used anywhere..do we even need it? 🤷‍♀️ 
https://github.com/search?q=org%3AFinancial-Times%20groupItemsByLocalisedDate&type=code